news/2024/2024-10-09-resurrection-cup-2024-results.md

```diff
@@ -0,0 +1,417 @@
+
+</div>
+
+## Staff interviews
+
+A lot of people were involved in making Resurrection Cup happen. We've asked a few more of those people to give insights into what makes Resurrection Cup... Resurrection Cup.
+
+::{#alvearia-and-luminous-sky}::
+
+![](/wiki/shared/news/2024-10-09-resurrection-cup-2024-results/alvearia-and-luminous-sky.jpg)
+
+<div class="news-chat-quote__text-container">
+
+<a class="avatar news-chat-quote__avatar" href="https://osu.ppy.sh/users/12840110" style="background-image: url('/wiki/shared/news/2024-10-09-resurrection-cup-2024-results/avatar-Phreel.jpg')"></a>
+
+<p class="news-chat-quote__username"><a class="news-chat-quote__colour-no-group" href="https://osu.ppy.sh/users/12840110">Phreel</a></p>
+
+What was your initial thought about having more than 20 original songs to do pooling with?
+
+<a class="avatar news-chat-quote__avatar" href="https://osu.ppy.sh/users/6248691" style="background-image: url('/wiki/shared/news/2024-10-09-resurrection-cup-2024-results/avatar-Alvearia.jpg')"></a>
+
+<p class="news-chat-quote__username"><a class="news-chat-quote__colour-no-group" href="https://osu.ppy.sh/users/6248691">Alvearia</a></p>
+
+I was surprised the most with the number of composers we were able to invite. When we had 5, 7, 10, I thought it's already too much and would require a lot of work to organize, but seeing more than 20 insanely talented people absolutely blew my mind. Each artist being unique with their genres and mixes, they still provided Resurrection Cup with astonishing songs that you would want to listen to on repeat. I'd say, my first initial thoughts were amazement and impatience to start the work on those projects.
+
+<a class="avatar news-chat-quote__avatar" href="https://osu.ppy.sh/users/4429612" style="background-image: url('/wiki/shared/news/2024-10-09-resurrection-cup-2024-results/avatar-Luminous_Sky.jpg')"></a>
+
+<p class="news-chat-quote__username"><a class="news-chat-quote__colour-no-group" href="https://osu.ppy.sh/users/4429612">Luminous Sky</a></p>
+
+While I was happy that we had lots of original songs, I knew these songs would hardly be the only custom maps we would end up with. Lo and behold, we had a total of 91 custom maps. 91!! That's an astonishingly large amount of customs and with our policy of trying not to reject a map due to slot-fitting reasons so long as the map itself is of acceptable quality, I knew pooling, or moreso the lack of it especially in later stages would make testing all skillsets, representing many genres and maintaining a similar difficulty across all maps in a pool quite the challenge. Regardless, I tried my best to cover missing skillsets as best as I could, sort of killing slot-pooling because of this, but only when necessary, like having a low BPM flow aim FM1 in Finals using Prayer for example.
```

Comment:
what a long sentence

```suggestion
While I was happy that we had lots of original songs, I knew these songs would hardly be the only custom maps we would end up with. Lo and behold, we had a total of 91 custom maps. 91!!

That's an astonishingly large amount of customs, and with our policy of trying not to reject a map due to slot-fitting reasons, so long as the map itself is of acceptable quality, I knew pooling, or moreso the lack of it, especially in later stages, would make testing all skillsets representing many genres and maintaining a similar difficulty across all maps in a pool quite the challenge.

Regardless, I tried my best to cover missing skillsets as best as I could, sort of killing slot-pooling because of this, but only when necessary, like having a low BPM flow aim FM1 in Finals using Prayer for example.
```